adapters/tappx/tappx.go

```diff
@@ -131,12 +131,12 @@ func (a *TappxAdapter) buildEndpointURL(params *openrtb_ext.ExtImpTappx, test in
 		}
 	}
 
-	tappxHost := "tappx.com"
-	isNewEndpoint, err := regexp.Match(`^(zz|vz)[0-9]{3,}([a-z]{2}|test)$`, []byte(params.Endpoint))
+
+	isNewEndpoint, err := regexp.Match(`^(zz|vz)[0-9]{3,}([a-z]{2,3}|test)$`, []byte(params.Endpoint))
 	if isNewEndpoint {
-		tappxHost = params.Endpoint + ".pub." + tappxHost + "/rtb/"
+		tappxHost = params.Endpoint + ".pub.tappx.com/rtb/"
 	} else {
-		tappxHost = "ssp.api." + tappxHost + "/rtb/v2/"
+		tappxHost = "ssp.api.tappx.com/rtb/v2/"
 	}
```

Comment:
There are two kinds of endpoints for this adapter. Can you standardize on one? In that choice, the "old endpoint" is preferred by Prebid.

Please see this thread: https://github.com/prebid/prebid-server/issues/2606

Reply:
Hello @SyntaxNode,
In short term we still need both, but we are planning at mid term to leave only one.
But the one we plan to leave is the "new endpoint", I know this is more "dynamic" but we need to adapt the endpoint to each Publisher in order to work correctly.